packages/dma-contracts/tasks/operations-registry/index.ts

```diff
@@ -0,0 +1,212 @@
+
+  const operationRegistry: OperationRegistryMaybe = await getOperationRegistry(
+    ethers.provider,
+    systemConfig,
+  )
+  if (!operationRegistry) {
+    console.log('OperationRegistry not deployed, cannot fetch values')
+    return
+  }
+
+  const property = getPropertyFromPath(config, configPath)
+  if (!property) {
+    throw new Error(`Property ${configPath} not found`)
+  }
+
+  console.log('\n====== REMOTE CONFIG ======')
+  console.log(`Network: ${network}`)
+  console.log(`Path: ${configPath}`)
+  console.log(`OperationsRegistry: ${operationRegistry.address}`)
+  console.log('==============================\n')
+
+  for (const operation of Object.keys(property)) {
+    const operationName: string = property[operation]
+    await validateOperationWithLog(
+      operationName,
+      operationRegistry,
+      operationsDatabase,
+      actionsDatabase,
+    )
+  }
+
+  console.log('\n==============================\n')
+}
+
+async function pushConfigToRemote(configPath: string, network: string, ethers: any) {
+  const signer = (await ethers.getSigners())[0]
+  const actionsDatabase: ActionsDatabase = new ActionsDatabase(network as Network)
+  const operationsDatabase: OperationsDatabase = new OperationsDatabase(network as Network)
+  const systemConfig: SystemConfig = getConfigByNetwork(network as Network) as SystemConfig
+  const config = OPERATION_NAMES
+
+  const operationRegistry: OperationRegistryMaybe = await getOperationRegistry(signer, systemConfig)
+  if (!operationRegistry) {
+    console.log('OperationRegistry not deployed, cannot fetch values')
+    return
+  }
+
+  const property = getPropertyFromPath(config, configPath)
+  if (!property) {
+    throw new Error(`Property ${configPath} not found`)
+  }
+
+  console.log('\n====== ADDING CONFIG ======')
+  console.log(`Network: ${network}`)
+  console.log(`Path: ${configPath}`)
+  console.log(`OperationsRegistry: ${operationRegistry.address}`)
+  console.log('==============================\n')
+
+  for (const operation of Object.keys(property)) {
+    const operationName: string = property[operation]
+
+    const validationResult = await validateOperationWithLog(
+      operationName,
+      operationRegistry,
+      operationsDatabase,
+      actionsDatabase,
+      true,
+    )
+
+    if (validationResult.type === OpValidationResultType.NOT_CONFIGURED) {
+      const operationDefinition: OperationDefinition | undefined =
+        operationsDatabase.getDefinition(operationName)
+      if (!operationDefinition) {
+        console.log(`[❌][ UNDEFINED ] ${operationName} is not defined in the local config`)
+        continue
+      }
+
+      const operationHashes: string[] = operationDefinition.actions.map(
+        (action: ActionDefinition) => action.hash,
+      )
+      const operationOptionals: boolean[] = operationDefinition.actions.map(
+        (action: ActionDefinition) => action.optional,
+      )
+
+      const tx = await operationRegistry.addOperation({
```

Comment:
its not behind multisig ?

Reply:
Yes, that is why there is a disclaimer in the Notion page about this tool being only valid for an initial deployment, when the deployer is still the owner. For the other functionality, we will need to integrate the Safe SDK, but we can definitely reuse this tools and add a flag to indicate whether we want to add the operations, or propose them in Safe

Reply:
ah ok ! I haven't read the notion page :)